dags/nocp/config/benchmarks/ocm.json

```diff
@@ -5,38 +5,11 @@
             "env": {
 		"OCM_TOKEN_URL": "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token",
 		"GATEWAY_URL": "https://api.integration.openshift.com",
-                "ES_INDEX": "ocm-request-test",
-		"RAMP_TYPE": "exponential",
-		"RATE": "10/s",
-		"RAMP_START_RATE": "1",
-		"RAMP_END_RATE": "120",
-		"RAMP_STEPS": "1",
-		"RAMP_TYPE": "exponential",
-		"GATEWAY_URL": "https://api.integration.openshift.com/",
 		"JOB_TIMEOUT": "28800",
-		"DURATION": "30",
 		"COOLDOWN": "60",
 		"SLEEP": "300",
-		"SELF_ACCESS_TOKEN_RATE": "5/s",
-		"LIST_SUBSCRIPTIONS_RATE": "5/s",
-		"ACCESS_REVIEW_RATE": "100/s",
-		"REGISTER_EXISTING_CLUSTER_RATE": "100/s",
-		"CREATE_CLUSTER_DURATION": "5",
-		"LIST_CLUSTERS_RATE": "20/s",
-		"GET_CURRENT_ACCOUNT_DURATION": "10",
-		"RESOURCE_REVIEW_RATE": "5/s",
-		"CLUSTER_AUTHORIZATIONS_RATE": "10/s",
-		"CLUSTER_AUTHORIZATIONS_DURATION": "15",
-		"SELF_TERMS_RATE": "30/s",
-		"SELF_TERMS_DURATION": "10",
-		"CERTIFICATES_RATE": "15/s",
-		"CERTIFICATES_DURATION": "10",
-		"CREATE_SERVICES_RATE": "1/m",
-		"CREATE_SERVICES_DURATION": "1",
-		"GET_SERVICES_RATE": "1/m",
-		"GET_SERVICES_DURATION": "1",
-		"PATCH_SERVICES_RATE": "1/m",
-		"PATCH_SERVICES_DURATION": "1",
+		"ORCHESTRATION_USER": "root",
+		"ORCHESTRATION_HOST": "airflow-ocm-jumphost.rdu2.scalelab.redhat.com",
```

Comment:
Can this be a vault var instead of being hard coded into our repo?

Reply:
thanks Russ, addressed it.